internal/sdkv2provider/resource_user_identity.go

```diff
@@ -46,6 +47,41 @@ func resourceUserIdentity() *schema.Resource {
 				}
 			},
 		},
+
+		Importer: &schema.ResourceImporter{
```

Comment:
We should use our new passthrough importer!

Reply:
I did that successfully (now it's commented out because I changed the identity version to manually test the upgrader)